osm.py

```diff
@@ -248,6 +262,10 @@ def main():
         elif args.update:
             ensure_valid_vault(vault_paths, args.update)
             call_for_each_vault(vault_paths, copy_settings, Path.home() / args.update, args.rm)
+        elif args.diff_to:
+            ensure_valid_vault(vault_paths, args.diff_to)
+            # Note: By setting DIFF_CMD above, we can re-use the copy_settings helper.
```

Comment:
This will change in the next commit, but it was easier to leave it here in this commit because I based this branch off the end of #2